src/pybel/manager/cache_manager.py

```diff
@@ -717,14 +717,69 @@ def drop_network_by_id(self, network_id):
         :param int network_id: The network's database identifier
         """
         network = self.session.query(Network).get(network_id)
+        edges = network.edges.all()
```

Comment:
can you explain what the logic is behind the changes to this function are in the documentation